github/Organization.py

```diff
@@ -477,6 +477,7 @@ def create_repo(
         allow_merge_commit=github.GithubObject.NotSet,
         allow_rebase_merge=github.GithubObject.NotSet,
         delete_branch_on_merge=github.GithubObject.NotSet,
+        visibility=github.GithubObject.NotSet,
```

Comment:
You should also edit the github/Organization.pyi file to add the new parameter.

Reply:
Ah yes. Thank you!